### DISCUSSION TITLE
Unicity and sort on pair fields

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15515] [Archive]
- `Table A` has a `Field_A`
- `Table B` has a `Field_B`
- and a `Table A_B` of pairs `(A, B)`
  - has a "Field_A" as M2O on `Table A.Field_A`
  - and a "Field_B" as M2O on `Table B.Field_B`

Only certain pairs are allowed (and they are created by the administrator)

Question  1: Is there a way to ensure the unicity of pairs in Table A_B?
Question  2: Is there a way to sort Table A_B on its (Field_A, Field_B)?

I can handle that directly on the DB (PostgreSQL), but is it possible to handle through Directus?

**Comment by br41nslug** [comment]
I don't think you can configure such composite unique indexes in Directus without extensions/flows checking the uniqueness on create.
